go/exorelay/features/sending.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Sending outgoing messages
+
+  As an Exoservice developer
+  I want my service to be able to send messages to other Exosphere services
+  So that it can interact with the rest of the application.
+
+  Rules:
+  - call "Send" on your ExoRelay instance have it send out the given message
+  - provide the message to send as the first parameter
```

Comment:
Let's refine this to: "provide the __name__ of the message to send as the first parameter"